### PR TITLE
Adding bug report button in footer

### PR DESCRIPTION
**Pull request by anhduy1202**
Feel free to leave any comments :)

### CODE REVIEW

File: src/lib/components/sections/footer.svelte [hunk 1]
```diff
@@ -10,10 +11,14 @@
 				acm<span class="brand-em brand-light">CSUF</span>
 			</span>
 		</p>
-
-		<a href="/github" class="github" target="_blank" rel="noopener norefferer">
-			<GitHubIcon />
-		</a>
+		<div class="footer-link">
+			<a href="/bug" class="bug" target="_blank" rel="noopener norefferer">
```
Comment (EthanThatOneKid) [inline src/lib/components/sections/footer.svelte]: Nitpicking here, but can we use `/bugs` instead of `/bug`?

File: src/lib/constants/links.ts [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
-  'youtube': "https://www.youtube.com/channel/UCDMBj0o4V8Cpt0jgsZLNVVg",
-  'instagram': "https://instagram.com/acmcsuf",
-  'discord': "https://tinyurl.com/acm-csuf-discord/", // Fallback: <https://discord.com/invite/Dfssdtd>
-  'linkedin': "https://linkedin.com/company/acm-at-csuf/",
-  'nbapp': "https://docs.google.com/forms/d/e/1FAIpQLSftpwDGyth2m9D7-_OpihrxoeqVyaRqhZp5edetWcg9KKsnYQ/viewform",
-  'tuffyhacks': "https://tuffyhacks.com/",
-  'acm-w': "https://sites.google.com/view/acm-w",
-  'devf21': "https://docs.google.com/forms/d/e/1FAIpQLSe53VPrM1myjU_BQQYQCw9MXJb9VYlSQ0U_xe8SRUE-I8PnBw/viewform",
+	github: 'https://github.com/EthanThatOneKid/acmcsuf.com',
+	bug: 'https://github.com/EthanThatOneKid/acmcsuf.com/issues/new?assignees=&labels=bug&template=bug_report.yaml',
```
Comment (EthanThatOneKid) [inline src/lib/constants/links.ts]:
```suggestion
	bugs: 'https://github.com/EthanThatOneKid/acmcsuf.com/issues/new?assignees=&labels=bug&template=bug_report.yaml',
```

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
Thanks for submitting this PR. We are currently discussing redesigning the footer, but I left some nitpicks in the meantime.

File: src/lib/components/sections/footer.svelte (hunk 1 above)
Comment (anhduy1202) [reply src/lib/components/sections/footer.svelte]: done!

**Pull request closed without merging**